TAF/utils/scripts/docker/get-compose-file-backward.sh

```diff
@@ -30,8 +35,19 @@ sed -n \
     -e '/metadata:/ r temp/core-services.yaml' \
     -e '/scheduler:/,/Service_Host: edgex-device-virtual/ p' \
     -e '/Service_Host: edgex-device-virtual/ r temp/device-virtual.yaml' \
-    -e '/#  device-random:/,$ p' \
+    -e '/ device-rest:/,$ p' \
     temp/geneva-compose.yaml > docker-compose-backward.yaml
 
-rm -rf temp
+# nightly build compose has ARCH variable for the image that geneva doesn't have
+if [ "$USE_ARCH" = "arm64" ]; then
+  sed -i 's/\${ARCH}/-arm64/g' docker-compose-backward.yaml
+else
+  sed -i 's/\${ARCH}//g' docker-compose-backward.yaml
+fi
 
+sed -i 's/\${CORE_EDGEX_REPOSITORY}/nexus3.edgexfoundry.org:10004/g' docker-compose-backward.yaml
+sed -i 's/\${CORE_EDGEX_VERSION}/master/g' docker-compose-backward.yaml
+sed -i 's/\${DEV}//g' docker-compose-backward.yaml
+sed -i 's/\- database/- redis/g' docker-compose-backward.yaml
```

Comment:
The database is started by compose name "database", not 'redis' on deploy-edgex.sh. This line should be removed.

Reply:
Ok, yea it needs `redis` service name depends all changed to `database` like in new compose. Fixed. :-)